static/app/views/alerts/rules/details/index.tsx

```diff
@@ -186,31 +186,36 @@ class AlertRuleDetails extends Component<Props, State> {
 
     return (
       <Projects orgId={organization.slug} slugs={rule?.projects}>
-        {({projects}) => (
-          <PageFiltersContainer
-            shouldForceProject
-            forceProject={projects[0] as Project}
-            forceEnvironment={rule?.environment ?? ''}
-            lockedMessageSubject={t('alert rule')}
-            showDateSelector={false}
-          >
-            <SentryDocumentTitle title={rule?.name ?? ''} />
-
-            <DetailsHeader
-              hasIncidentRuleDetailsError={hasError}
-              params={params}
-              rule={rule}
-            />
-            <DetailsBody
-              {...this.props}
-              rule={rule}
-              project={projects[0] as Project}
-              incidents={incidents}
-              timePeriod={timePeriod}
-              selectedIncident={selectedIncident}
-            />
-          </PageFiltersContainer>
-        )}
+        {({projects}) => {
+          const project = projects.find(({slug}) => slug === rule?.projects[0]) as
+            | Project
+            | undefined;
+          return (
+            <PageFiltersContainer
+              shouldForceProject
```

Comment:
I think there might be problems if you pass `shouldForceProject` and an undefined `project`.

Reply:
maybe we can use isGlobalSelectionReady

Reply:
okay, figured out the loading state